Sources/Core/Extensions/String+Extensions.swift

```diff
@@ -18,20 +18,33 @@
 
 import Foundation
 
-// This will eventually be replaced by the rewrite of FBSDKAccessTokenCache
-// for now it is needed as a transient dependency of AccessTokenWallet (via Settings)
+extension String {
+  /**
+   Converts a camelCase string to snake_case
 
-// TODO:
-// This public, canonical type makes this usable from anywhere that knows about access tokens.
-// Internally, it should have a bi-directional codable "mirror type", something like
-// `CacheableAccessToken` that is used for converting a canonical AccessToken to/from a
-// serializable medium. This will help separate the AccessToken type from any specific
-// encoding formats.
-//
-class AccessTokenCache: AccessTokenCaching {
-  var accessToken: AccessToken?
+   - Returns: The string converted to snake case
+   */
+  func snakeCased() -> String? {
```

Comment:
Why go with a function over a computed variable?

Reply:
Similar to [`lowercased()`](https://developer.apple.com/documentation/swift/string/1641392-lowercased) in the String function.